tests/unit/services/test_init.py

```diff
@@ -218,10 +282,48 @@ def test_render_project_executable(init_service, tmp_path):
         environment=environment,
         project_dir=project_dir,
         template_dir=template_dir,
-        context={"name": "my-project"},
+        context={"name": "my-project", "version": init_service.default_version},
     )
 
     pytest_check.is_true(os.access(project_dir / "file-1.sh", os.X_OK))
     pytest_check.is_true(os.access(project_dir / "file-2.sh", os.X_OK))
     pytest_check.is_false(os.access(project_dir / "file-3.txt", os.X_OK))
     pytest_check.is_false(os.access(project_dir / "file-4.txt", os.X_OK))
+
+
+def test_initialise_project(
```

Comment:
I originally wrote something similar to this and scrapped it in favor of more integration tests because it feels like this only tests that mocked private functions are called.

Reply:
I did it only to satisfy `codecov` uncoveraged patch line  :sweat_smile:

Reply:
That reminds me, I'm going to add a discussion point for code coverage in integration tests to the next engineering sync.  I think we were waiting on Alex to return to get his opinion.